src/GraphQL.Tests/Validation/VariablesAreInputTypesTests.cs

```diff
@@ -38,10 +38,6 @@ public void output_types_are_invalid()
                 message: VariablesAreInputTypesError.UndefinedVarMessage("c", "Pet"),
                 line: 1,
                 column: 40);
-            _.Error(
-               message: "Variable '$b' is invalid. No value provided for a non-null variable.",
-               line: 1,
-               column: 20);
```

Comment:
Again, this error is really inaccurate; variable $b was correctly and already reported as having an invalid type.  It should not report again as being not provided.